int32 signal
---
bool return